Makefile

```diff
@@ -9,7 +9,13 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+vet:
+	go list ./... | grep -v "./vendor*" | xargs go vet
```

Comment:
No, What i mean is that i want to exclude node-problem-detector binary file under root directory. I forgot that it does not need a dot prefix. I have confirmed this according to  http://stackoverflow.com/a/3637678/1889327